baseapp/baseapp.go

```diff
@@ -682,20 +689,36 @@ func (app *BaseApp) runMsgs(ctx sdk.Context, msgs []sdk.Msg, mode runTxMode) (*s
 			break
 		}
 
-		msgRoute := msg.Route()
-		handler := app.router.Route(ctx, msgRoute)
+		var msgEvents sdk.Events
+		var msgResult *sdk.Result
+		var err error
+		var msgType string
+
+		if svcMsg, ok := msg.(sdk.ServiceMsg); ok {
+			msgType = svcMsg.MethodName
```

Comment:
Field is called MethodName yet the var is called msgType.

Reply:
Renamed to `msgFqName`